packages/api/tests/endpoints/test-reconciliation-reports.js

```diff
@@ -198,7 +237,9 @@ test.serial('get a report', (t) =>
       .set('Accept', 'application/json')
       .set('Authorization', `Bearer ${jwtAuthToken}`)
       .expect(200);
-    t.deepEqual(response.body, { test_key: `${record.name} test data` });
+    t.deepEqual(response.body, {
```

Comment:
This would be easier to read if it were:

```js
t.deepEqual(
  response.body,
  { test_key: `${record.name} test data` }
);
```

or

```js
t.deepEqual(
  response.body,
  {
    test_key: `${record.name} test data`
  }
);
```